### ISSUE TITLE
Features list loads slowly and blocks rendering

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/chromium-dashboard#1915]
Thank you for letting us know that something needs to be fixed. This page is for recording bugs on Chrome Status (https://www.chromestatus.com). If you need to report a problem with Chrome or Chromium, please use crbug (http://crbug.com/new). Thank you.

**Describe the bug**
The features.json file loads slowly, sometimes taking as long as 30 seconds or more.

**To Reproduce**
1. Open Chrome Status.
2. Reload the page.

If it happens to no load slowly:
1. Open Chrome Status.
2. Open DevTools.
3. Click "Empty Cache and Hard Reload."

**Expected behavior**
I expect its load time to be comparable to resources on the same page, with the same size.

**Additional context**
<img width="963" alt="Screen Shot 2022-06-06 at 11 10 40 AM" src="https://user-images.githubusercontent.com/8204171/172190429-0776394c-1945-48c6-96b7-618018a1d73a.png">

**Comment by jrobbins** [comment]
The existing feature list has several problems, including slow performance because of requesting features.json which is a huge file.   The long-term replacement is the new feature page which does server-side searches and returns paginated results.  That page is still a work in progress.

**Comment by jyasskin** [comment]
The new page is currently slower than the old page, with a 7s delay on one server-side query:
![Chrome devtools network timing output, showing 7s of "Waiting for server response" on a request for features?q=first&num=100](https://user-images.githubusercontent.com/83420/211384896-f5e71f16-2503-4d1a-a59e-c34305ab70c7.png)

**Comment by jrobbins** [comment]
Yeah, something got slower.   I have an idea of what the slow part might be.
